policies/queries/ec2/internet_gateways_unused.sql

```diff
@@ -0,0 +1,14 @@
+insert into aws_policy_results
+with attachment as (select distinct internet_gateway_cq_id from aws_ec2_internet_gateway_attachments)
+select :'execution_time'                            as execution_time,
+       :'framework'                                 as framework,
+       :'check_id'                                  as check_id,
+       'Unused internet gateways should be removed' as title,
+       gateway.account_id,
+       gateway.id                                   as resource_id,
+       case
+           when attachment.internet_gateway_cq_id is null then 'fail'
+           else 'pass'
+           end                                      as status
+from aws_ec2_internet_gateways gateway
+         left join attachment on attachment.internet_gateway_cq_id = gateway.cq_id
```

Comment:
Is it possible to create an IGW and not attach it to a VPC?

Reply:
https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Internet_Gateway.html#Add_IGW_Attach_Gateway
https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateInternetGateway.html
apparently, yes